src/how/NQPParametricRoleHOW.nqp

```diff
@@ -105,6 +106,14 @@ knowhow NQPParametricRoleHOW {
 
     # Compose the role. Beyond this point, no changes are allowed.
     method compose($obj) {
+        @!role_typecheck_list := my @rtl;
```

Comment:
Why are you using this `my @rtl` instead of just pushing directly to `@!role_typecheck_list`?

Reply:
At this point, `@!role_typecheck_list` hasn't been bound yet. Should probably be bound sooner to avoid `role_typecheck_list` returning a `VMNull` before composition.

Reply:
Ah, then what about `@!role_typecheck_list := nqp::list();` up in `BUILD()`?

Reply:
That works.